flutter/multi_user_pub.dart

```diff
@@ -0,0 +1,2 @@
+final channel = ably.channels.get('doc:mars-launch-plan');
+await channel.publish('update', {'pos': chatAt, 'insert': 'Send Grimes first'});
```

Comment:
@fliptopbox can you advise on max practical width in the design?  Should this be on three lines or one for example, the guideline width in terms of chars will help.

Reply:
- the absolute max-width before the code will wrap is 64 characters
- the recommended length is 60 characters

for this example .... extract the Ojbect literal
```
const options = {'pos': chatAt, 'insert': 'Send Grimes first'};
await channel.publish('update', options);
```

@mattheworiordan @jamienewcomb

Reply:
Can we please state this somewhere that can be seen by all and communicate it to the developers working on these snippets? Even the JavaScript version has code with lines longer than 64 chars (i.e. 69 in [gps-pub.js](https://github.com/ably/homepage-snippets/blob/main/javascript/gps-pub.js) and 72 in [multi-user-pub.js](https://github.com/ably/homepage-snippets/blob/main/javascript/multi-user-pub.js)), and that's what we're currently using as the reference.

Reply:
@lmars yup. it now added it to the README. https://github.com/ably/homepage-snippets

Reply:
conform to line width: bce771d